src/pretix/presale/checkoutflow.py

```diff
@@ -700,6 +702,18 @@ def post(self, request, *args, **kwargs):
                        sales_channel=request.sales_channel.identifier)
 
 
+def business_heuristic(organizer):
```

Comment:
I also thought about it and I'm not sure which is better, therefore I just kept it the way Raphael did in his example. Advantage of using the organizer is that if the organizer has previous events, there is data immediately for new events. One could add a condition to only use the organizer's data if the event has too few orders.

Reply:
Yep, that might be the right middle-way. Use the event’s orders if we have enough (whatever that is), else use any orders for the organizer (whether all or just the last X – see discussion below).

Reply:
Exactly, I suggested organizer so we don't have confusion because the logic resets everytime a new event is created. But yes, there are some organizers who have separate events that are heavily business or heavily personal.

So yeah, maybe "use event data if there are more than 100 orders on the event, otherwise use organizer data"?